bigquery/README.md

```diff
@@ -9,6 +9,13 @@ analytics data warehouse.
 
 ## Samples
 
+### Quickstart
+
+'''
```

Comment:
That is a incorrectly typed symbol. (I meant to use an apostrophe). I since have removed them, because Remi and I decided not to have them in the README for now.